app/views/profiles/_form.html.erb

```diff
@@ -29,7 +29,13 @@
       </div>
       <div class="col-span-full">
         <%= form.label :disciplines,  class: "block text-sm font-medium leading-6 text-gray-900" %>
-        <%= form.text_field :disciplines, class: "block w-full max-w-96 lg:w-1/3 rounded-md border-0 py-1.5 text-gray-900 ring-1 ring-inset ring-gray-400 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-500 sm:text-sm sm:leading-6" %>
+        <div class="flex flex-wrap my-3">
+        <%= form.collection_check_boxes :tag_ids, Tag.all.order(:name), :id, :name do | tag | %>
+          <span class="w-1/2 lg:w-1/4">
+              <%= tag.label(class: "mr-2 text-sm font-medium leading-6 text-gray-900") { tag.check_box(class: "mr-2") + tag.text } %>
+          </span>
+        <% end %>
+        </div>
```

Comment:
Just like everything in rails it took me a while to figure this out but when it works it is magical